### ISSUE TITLE
Ruby: Pry does not handle non-ASCII bytes

### ISSUE THREAD

**Issue by vainikkaj** [termux/termux-packages#262]
I'm suspecting that this is somehow Termux-related because I've installed 2.3.1 using RVM and following snippet works there.

```
2.3.1 (main):0 > show-source Pry
ArgumentError: invalid byte sequence in US-ASCII
from /data/data/com.termux/files/usr/lib/ruby/gems/2.3.0/gems/method_source-0.8.2/lib/method_source/code_helpers.rb:75:in `=~'
```

**Comment by fornwall** [comment]
This is now fixed in version `2.3.1-3` of the ruby package, now available as an update.

Thanks for helping to improve the ruby Termux package :)!

**Comment by vainikkaj** [comment]
For a project as awesome as Termux, it's the least I can do =)

And as always, thanks for a quick fix.
